src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -322,67 +176,58 @@ def static_dag(job,
     work_dir = job.fileStore.getLocalTempDir()
 
     # what region is our bucket in?
-    if bucket_region == "us-east-1":
-        bucket_region = ""
+    if inputs.bucket_region == "us-east-1":
```

Comment:
Unrelated to your change, but I scratched my head for a bit when reading this line change. Can you add to the comment above that this is necessary for special casing the S3 http URLs?

Actually, do we still use the s3 http URLs or has everything cut over to s3:// URLs? If we've fully moved over—and I think we have—then we can remove this code/config option, IIRC.